File: Basic/10.Swap two numbers without using a third variable/Solution.py
if __name__ == "__main__":
    a = int(input("Enter the first number (a): "))
    b = int(input("Enter the second number (b): "))

    print(f"Before swapping: a = {a}, b = {b}")

    a = a + b
    b = a - b
    a = a - b

    print(f"After swapping: a = {a}, b = {b}")

File: Basic/10.Swap two numbers without using a third variable/Solution.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b;

    cout << "Enter the first number (a): ";
    cin >> a;
    cout << "Enter the second number (b): ";
    cin >> b;

    cout << "Before swapping: a = " << a << ", b = " << b << endl;

    a = a + b;
    b = a - b;
    a = a - b;

    cout << "After swapping: a = " << a << ", b = " << b << endl;

    return 0;
}
